action.yml

```diff
@@ -34,11 +41,19 @@ runs:
         --extensions=${{ inputs.lint }} ${{ inputs.path }}
     - name: Standards
       shell: bash
+      if: always()
       run: >
         ${{ github.action_path }}/vendor/bin/phpcs -s --standard=Drupal,DrupalPractice
-        --extensions=${{ inputs.extensions }} ${{ inputs.path }} --ignore=*.md
-    - name: Copy-Pasta
+        --extensions=${{ inputs.extensions }} ${{ inputs.path }} --ignore=${{ inputs.phpcs-ignore }}
+    - name: Copy-Pasta - Exclude
+      shell: bash
+      if: ${{ always() && inputs.phpcpd-exclude != '' }}
+      run: >
+        ${{ github.action_path }}/vendor/bin/phpcpd --suffix=${{ inputs.suffix }}
+        --exclude ${{ inputs.phpcpd-exclude }} ${{ inputs.path }}
```

Comment:
Generally a good idea to be consistent with how options are specified, like: including or excluding the `=` kind of thing... unless it isn't actually accepted here?

Reply:
Couldn't find consistent documentation on using either `exclude` or `exclude=`, after digging into the actual code looks like `exclude=` is used and `=` is used in other options, so including it for consistency.